robottelo/utils/vault.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, env_file='.env'):
     def setup(self):
         if self.env_path.exists():
             self.envdata = self.env_path.read_text()
-            is_enabled = re.findall('\nVAULT_ENABLED_FOR_DYNACONF=(.*)', self.envdata)
+            is_enabled = re.findall('VAULT_ENABLED_FOR_DYNACONF=(.*)', self.envdata)
             if is_enabled:
                 self.vault_enabled = is_enabled[0]
```

Comment:
`self.vault_enabled = is_enabled[-1]`

Reply:
first or last  - what is the difference when you expect just single ocurrence

when it is duplicate the last would override prevoius ones .... hrm

Reply:
it's just a good practice to use the last value if there are multiple entries of one key. but it's best practice to don't repeat keys in config files.

the original problem (unable to read value when key is on first line) is fixed - approved